### PR TITLE
#911 What is your emergency?

### LINKED ISSUE #911: Don't allow FDA members to select "No Org" in selector
### Back-end:

- add `isSystemOrg` boolean field to "organisation" table (and set "true" for "Food and Drug agency" org)
- return this field as part of OrgInfo from '/userInfo' endpoint

### Front-end:

- check `isSystemOrg` field in OrgSelector, and only display "No Org" option if false
- in Login screen, don't present  org selection option if `isSystemOrg`

Anything else?

### PR DESCRIPTION
**Pull request by CarlosNZ**
Fixes #911 

Requires [back-end PR #524](https://github.com/openmsupply/application-manager-server/pull/524)

This is kind of an interim solution, we can come up with something else later on if it's a problem. For now, the FDA org never see the "No Org" selection, which is what Craig wants.

- Also tweaked the logic on the login screen, so if a user only has one org (i.e. most people), they'll skip the org selection and automatically be logged in as their only org (so won't see the "No Org" option initially). So only people who go poking around in the Org dropdown will see the "No Org" option

To test, just set the "isSystemOrg" flag on any org to "true" and note that users who are part of this org can no longer see the "No Org" option

Edit: improved it further so there is not even a Dropdown at all when there is only one choice (i.e one company, FDA), it's just a fixed banner)

### CODE REVIEW

File: src/containers/User/UserArea.tsx [hunk 1]
```diff
@@ -191,22 +191,28 @@ const OrgSelector: React.FC<{ user: User; orgs: OrganisationSimple[]; onLogin: F
...
       )}
       <div>
-        <Dropdown
-          text={user?.organisation?.orgName || strings.LABEL_NO_ORG}
-          options={dropdownOptions}
-          onChange={handleChange}
-        ></Dropdown>
+        {dropdownOptions.length === 1 ? (
+          user?.organisation?.orgName || strings.LABEL_NO_ORG
+        ) : (
```
Comment (nmadruga) [inline src/containers/User/UserArea.tsx:209]: What happens when lenght === 0?

It seems like this statement should be <=1

**Review by nmadruga — CHANGES_REQUESTED** [review]
Seems really good. Quick testing now, will let you know if something unexpected happens

File: src/containers/User/UserArea.tsx (hunk 1 above)
Comment (CarlosNZ) [reply src/containers/User/UserArea.tsx:209]: The whole "orgSelector" component won't render if it's 0. See line 37:
```
{orgList.length > 0 && <OrgSelector user={currentUser} orgs={orgList} onLogin={onLogin} />}
```

Comment (nmadruga) [reply src/containers/User/UserArea.tsx:209]: Oh! Great. I tested too and it seems all good.

**Review by nmadruga — APPROVED** [review]
LGTM

**Comment by CarlosNZ** [comment]
> LGTM

Thanks -- back-end okay too? (https://github.com/openmsupply/application-manager-server/pull/524)

**Pull request merged**